books/RayTracingTheRestOfYourLife.html

```diff
@@ -705,15 +1182,14 @@
 Let’s review now because that was most of the concepts that underlie MC ray tracers.
```

Comment:
I know it's not part of your changes, but how about rewording this to “That was most of the concepts that underlie Monte Carlo ray tracers. Let's review:”

Reply:
Oops. “Those were most of the concepts ...”